### DISCUSSION TITLE
Styling the <IconItem> component?

### DISCUSSION THREAD

**Discussion by pedaars** [storybookjs/storybook#24697] [General]
I am trying to use the IconGallery and IconItem components to build a gallery of our used icons but I am unable to style the IconItem to prevent text overflow

![Screenshot 2023-11-03 at 08 49 10](https://github.com/storybookjs/storybook/assets/11647950/751ee4b8-94a2-451e-9dbd-5fc667d377a6)

**Comment by pedaars** [comment]
is this not possible?

**Reply by janniknilsson** [reply]
Late response, but you can add a style tag to the IconItem: style={{ flexDirection: 'column' }}.  It would be nice if you could set a style for all IconItems, instead of having to set it on each of them.

**Reply by koskosergej** [reply]
looks like Storybook has open issue for that case https://github.com/storybookjs/storybook/issues/24764

**Comment by koskosergej** [comment]
> Late response, but you can add a style tag to the IconItem: style={{ flexDirection: 'column' }}. It would be nice if you could set a style for all IconItems, instead of having to set it on each of them.

Unfortunately, the `style` tag ignored
